cmake/toolchain/xcc/common.cmake

```diff
@@ -7,7 +7,24 @@ if(NOT EXISTS ${XTENSA_TOOLCHAIN_PATH})
   message(FATAL_ERROR "Nothing found at XTENSA_TOOLCHAIN_PATH: '${XTENSA_TOOLCHAIN_PATH}'")
 endif()
 
-set(TOOLCHAIN_HOME ${XTENSA_TOOLCHAIN_PATH}/$ENV{TOOLCHAIN_VER}/XtensaTools)
+if(DEFINED ENV{TOOLCHAIN_VER})
```

Comment:
to have some consistency in Zephyr, can you update this to use `zephyr_get()` ?

Using `zephyr_get()` allows this setting to be obtain both as environment variable but also as CMake setting, using `-DTOOLCHAIN_VER=<value>`.

Furthermore it ensures that if the value is set in sysbuild, then it will propagate correctly to images.

Reply:
Done.